lib/content-services/document-list/components/document-list.component.ts

```diff
@@ -834,7 +836,8 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
     }
 
     private resetNewFolderPagination() {
-        this.pagination.value.skipCount = 0;
+        this._pagination.skipCount = 0;
+        this._pagination.maxItems = this.maxItems;
```

Comment:
this reset of the maxItems was not done until now and it would cause issues, when the 'Items per page' are changed by the user

Reply:
why?